tools/library/filter/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+message( STATUS "---------Beginning cmake build of ld-filter---------")
+project( ld-filter )
+
+#add_compile_options(-std=c++17
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread -O3 -Wall -ftree-vectorize -ffast-math -funroll-loops")
```

Comment:
It's not a good idea to hardcode compiler flags like this, but especially don't build ld-decode with `-ffast-math` - that makes maths operations faster at the cost of giving less accurate results!